.lagoon.yml

```diff
@@ -89,9 +111,27 @@ tasks:
     - run:
         name: Enable example content
         command: |
-          set -e
+          source dev-scripts/lagoon-error-handling.sh
+
           drush en -y dpl_example_content
         service: cli
+    - run:
+        name: Setting Deployment status success
+        command: |
+          DEPLOYMENT_STATUS="success"
+          DRUPAL_URL=$(drush browse)
+
+          # Read the deployment ID from the file
+          GH_DEPLOYMENT_ID=$(cat /tmp/gh_deployment_id)
+          echo "Setting GH deployment status '$GH_DEPLOYMENT_ID': '$DEPLOYMENT_STATUS'"
+
+          # Setting the deployment status, on the GH deployment created earlier.
+          curl -L \
+            -X POST \
+            -H "Authorization: Bearer $GH_DEPLOYMENT_TOKEN" \
+            https://api.github.com/repos/danskernesdigitalebibliotek/dpl-cms/deployments/$GH_DEPLOYMENT_ID/statuses \
+            -d "{\"environment\":\"$LAGOON_PR_HEAD_BRANCH\",\"state\":\"$DEPLOYMENT_STATUS\", \"target_url\":\"$DRUPAL_URL\"}"
```

Comment:
<img width="2552" alt="Monosnap Enable opening links in a new tab for `simple_links` paragraph + Update default content by kasperbirch1 · Pull Request #1617 · … 2024-10-03 09-53-56" src="https://github.com/user-attachments/assets/8218407e-0ce3-44f4-9a72-7029852f9603">

You probably need to reconsider your use of [`target_url` (or `log_url`) and `environment_url`](https://docs.github.com/en/rest/deployments/statuses?apiVersion=2022-11-28#create-a-deployment-status) when creating an environment status.